Gems/Atom/Feature/Common/Code/Include/Atom/Feature/PostProcess/EditorModeFeedback/EditorModeFeedbackInterface.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <Atom/Feature/Mesh/MeshFeatureProcessorInterface.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        //! The interface for the visual feedback component of the central editor mode tracker for all viewports.
+        class EditorModeFeedbackInterface
```

Comment:
question: Is this something that will move in future? Just curious as I think this has been mentioned to be following after the initial changes.

Reply:
Almost certainly. As to its final home, we're not 100% settled yet but it is an ongoing discussion that will be resolved in the next iteration 👍

Reply:
Fantastic, that seems like a sensible place to put it👍